src/lib/NotionSDK.ts

```diff
@@ -132,7 +132,7 @@ export class NotionSDK implements INotionSDK {
         }
     }
 
-    private async getPageObjectFromResults(item): Promise<IPage | null> {
+    private async getPageObjectFromResults(item, emoji:boolean=false): Promise<IPage | null> {
```

Comment:
As you are making changes in this file with suggestions, make a indentation with other methods too which has default value of emoji.
```suggestion
    private async getPageObjectFromResults(item, emoji:boolean = false): Promise<IPage | null> {
```